flatpak-warning.txt

```diff
@@ -19,8 +19,13 @@ To make the Integrated Terminal automatically use the host system's shell,
 you can add this to the settings:
 
   {
-    "terminal.integrated.shell.linux": "/usr/bin/env",
-    "terminal.integrated.shellArgs.linux": ["--", "flatpak-spawn", "--host", "bash"]
+    "terminal.integrated.defaultProfile.linux": "bash",
+    "terminal.integrated.profiles.linux": {
+        "bash": {
+          "path": "/usr/bin/env",
```

Comment:
I suspect that `env` here does nothing. Maybe invoke `/usr/bin/flatpak-spawn` directly instead?

Reply:
Yeah, works the same, let me fix that.